content/docs/contributing/to-docs/blog-posts.md

```diff
@@ -0,0 +1,106 @@
+Here's some basic Do's and Don'ts to help you scope your effort and avoid an obvious rejection.
+
+**Do:**
+
+* Discuss O3DE! (Duh.) Go deep! Highly technical blog posts, in particular, establish community credibility and provide a resource for people looking for deeper knowledge beyond the tech docs.
+
+* Discuss algorithms, techniques, technologies, and tools that apply directly to O3DE work. Again, not only is your knowledge and experience useful to capture as a published topic, but it helps the larger industry assess the caliber of the people building O3DE.
+
+* O3DE strategy, **if** you are actively participating in a SIG around feature or product strategy, **and** have the approval of a SIG.
+
+* Interviews with, or profiles of, important and established community members. People put a human face on a technical product, and the world reliably likes to hear humans say they have to say. In this case, it should be professional observations, knowledge, and commentary&mdash;and shuld be specifically related to O3DE. Personal opinions, as ever, should be largely technical.
+
+* Write with strong grammar and an eye for easy readability. Rambling monologues may get a gentle request for editorial review and a number of suggestions to "tighten it up". The reader's time is precious, so compressing your thoughts into short, information-dense sentences with regular paragraph breaks is just respectful.
+
+* Include images and diagrams wherever appropriate.
+
+* Consider your audience, which is diverse across many sociocultural and industry axes. Choose your words with respect and care, and be prepared to take feedback sincerely and make changes. No-one knows everything about the folks in the community and the industry, so don't let some initially careless wording caught in review bring you down; just make the changes, internalize the feedback, and keep delivering great content!
+
+**Don't:**
+
+* Get personal. We have no tolerance for using the O3DE blog to call out community individuals&mdash;or any individuals, really&mdash;in any negative way, no matter how concerned you are.
+
+* Get political. Politics pervade every part of life, and we all have different backgrounds, ideologies, and reactions to the way in which culture and society shapes itself. The O3DE blog isn't the place for it, unless you are running a community-approved political initiative and have the blessings of the Linux Foundation.
+
+* Attack or criticize other products, platforms, or companies directly. Showing perf data between O3DE and a similar product/feature pairing is one thing; slagging on a similar product or another company is another. The latter will not be tolerated, regardless of how strongly you feel on the subject.
+
+* Get too egotistical or too hyped. Choose your modifiers and adverbs with care to not oversell (or undersell) anything. Also, while a blog series can really help your resume (if done thoughtfully), the O3DE blog is ultimately about, well, O3DE. Not you.
```

Comment:
```suggestion
* Get too egotistical or too hyped. Choose your modifiers and adverbs with care to not oversell (or undersell) anything. Also, while a blog series can really help your resume (if done thoughtfully), the O3DE blog is ultimately about, well, O3DE. Not you.
```
Suggested rewrite: Exaggerate or overstate your points. Choose your modifiers and adverbs carefully to not oversell, or even undersell, anything. Also, while a blog series can really help your resume (if you do it thoughtfully), the O3DE blog is ultimately about, well, O3DE.